tracing/tracer.go

```diff
@@ -64,7 +64,10 @@ func (c Config) traceProvider(exporter func() (sdktrace.SpanExporter, []sdktrace
 
 	options := append([]sdktrace.TracerProviderOption{
 		sdktrace.WithSampler(sdktrace.AlwaysSample()),
-		sdktrace.WithSyncer(exp),
+		sdktrace.WithBatcher(exp,
+			sdktrace.WithMaxQueueSize(1000),
+			sdktrace.WithMaxExportBatchSize(1000),
```

Comment:
There are default values for queue size (2048) and batch size(512) as mentioned [here](https://pkg.go.dev/go.opentelemetry.io/otel@v0.2.3/sdk/trace#BatchSpanProcessorOption)

Is there a specific reason to set them to 1000?

Thx for the PR!

Reply:
Not really, I just based my code on what the uptrace example ( the provider I wanted to send traced to)

Reply:
I removed them to use the defaults